chapters/en/slides/module5/module5_03.Rmd

```diff
@@ -0,0 +1,316 @@
+    alt.X('date', title=None),
+    alt.Y('sum', axis=alt.Axis(format='$s'), title='Amount donated per day', scale=alt.Scale(zero=False)))
+```
+
+Notes:
+If we wanted to use the full range,
+we could set the domain explicitly like before,
+but the shortcut `zero=False` allows us to adjust the axis
+to the min and max values of the data with less typing.
+
+---
+
+## Large values can hide variation among the rest of the data
+
+```{python}
+alt.Chart(donations, title='Wikipedia donations peak during Christmas').mark_circle().encode(
+    alt.X('date', title=None),
+    alt.Y('sum', axis=alt.Axis(format='$s'), title='Amount donated per day'))
+```
+
+Notes:
+So far,
+we have seen how we can zoom in 
+to reveal detail of a specific region of the chart.
+
+What if we want to show all the chart,
+but still be able to see details for the days with lower total donations?
+
+---
+
+## Altair allows us to interactively zoom in and out of the plot
+
+```{python}
+alt.Chart(donations, title='Wikipedia donations peak during Christmas').mark_circle().encode(
+    alt.X('date', title=None),
+    alt.Y('sum', axis=alt.Axis(format='$s'), title='Amount donated per day')).interactive()
+```
+
+Notes:
+One of Altair's stong suites
+is that it is not limited to static visualizations.
+
+By simply appending the `interactive()` method to our chart 
+we can use the mouse to zoom by scrolling and pan by clicking and dragging!
+
+This allows us to interactively explore the data
+and zoom in on the points of interest as needed.
+We will learn more about interactive visualizations in the last module,
+this is just a sample of what is possible.
+
+How can we solve the problem of a large axis range in a static figure?
+
+---
+
+## A log transformed axis can reveal detail across a large range of values
+
+```{python}
+alt.Chart(donations).mark_point().encode(
+    alt.X('date'),
+    alt.Y('sum', axis=alt.Axis(format='$'), scale=alt.Scale(type='log')),
+    alt.Tooltip('week_day'))
+```
+
+Notes:
+By applying a logarithmic (log) transformation to the y-axis,
+we're able to increase the range used for the smaller values
+by compressing the space used for larger values.
+
+This means that log transformations are useful
+when there are large differences in the data values,
+such as in our case here.
+
+Log scales are also useful to show data that has been obtained from multiplication or division,
+such as ratios.
+
+The default log transformaton in Altair uses the base 10
+which is the most common choice.
+Every step on a log scale is referred to
+as a difference of an "order of magnitude".
+
+It is important to remember that we log scaled axes
+are not as easy to interpret as linear ones,
+and it is easy to underestimate actual the difference between large and small values.
+
+This is especially true
+when creating figures for communication
+with the general public
+rather than in a subject area
+where participants might be more used
+to seeing this transformation.
```

Comment:
Maybe add a sentence here saying something like, so if you are ever in this situation, make sure you clearly label your axes with the scaling and add some supporting text to help others correctly interpret the data with the transformed axes.